core/spec/models/spree/admin_user_spec.rb

```
@@ -53,4 +53,23 @@
+
+    context 'admin user invited other' do
+      let(:other_user) { create(:user, email: 'other_user@example.com') }
+      let(:invitation) { create(:invitation, email: other_user.email, inviter: admin_user) }
+
+      context 'other users accept invitation' do
+        before do
+          invitation.accept!
+        end
+
+        it 'does not remove other user\'s role' do
+          expect { destroy_admin_user }.to_not change { other_user.role_users.count }
```

RuboCop (1.81.6)

[convention] Prefer `not_to` over `to_not`.

(RSpec/NotToNot)